src/main/java/duke/command/AddDeadlineCommand.java

```diff
@@ -1,3 +1,8 @@
+package duke.command;
```

Comment:
Really great use of packaging by going beyond the single-package requirement and packaging different components of the final product instead.